src/Core/Common/FindMaxAndMinInImage/CMakeLists.txt

```diff
@@ -5,9 +5,11 @@ project(FindMaxAndMinInImage)
 find_package(ITK REQUIRED)
 include(${ITK_USE_FILE})
 
+find_package( VTK REQUIRED )
+include_directories(${VTK_INCLUDE_DIRS})
```

Comment:
You don't need this anymore; VTK's targets now have the include directories as part of their usage requirements already.